src/region/RegionCreationContainer.tsx

```diff
@@ -19,16 +21,24 @@ import withProviders from '../common/withProviders';
 export type Props = {
     isCreating: boolean;
     isDiscoverabilityEnabled: boolean;
+    isFtuxCursorDisabled: boolean;
     isRotated: boolean;
     staged: CreatorItemRegion | null;
 };
 
-export const mapStateToProps = (state: AppState, { location }: { location: number }): Props => {
+export const mapStateToProps = (
+    state: AppState,
+    { fileType, location }: { fileType?: string; location: number },
+): Props => {
     const staged = getCreatorStagedForLocation(state, location);
 
+    const isFtuxCursorDisabled =
```

Comment:
Individual annotation types should not be aware of the file type they're being created on. We should delegate all file-type and viewer-specific logic to the annotators.